packages/amplify-category-auth/provider-utils/awscloudformation/triggers/PostConfirmation/function-template-dir/trigger-index.js

```diff
@@ -3,11 +3,15 @@
   provided that the file names (without extension) are included in the "MODULES" env variable.
   "MODULES" is a comma-delimmited string.
 */
+const moduleNames = process.env.MODULES.split(',');
+const modules = [];
+for (let i = 0; i < moduleNames.length; i += 1) {
+  modules.push(require(`./${moduleNames[i]}`));
+}
```

Comment:
nit - this can simplify to:
```suggestion
const modules = moduleNames.map(name => require(`./${name}`));
```

Reply:
It's the first time I see/use the feature "Commit suggestion", it's a very good idea!